pipeline-steps/maas.groovy

```diff
@@ -0,0 +1,67 @@
+def prepare(Map args) {
+  common.conditionalStage(
+    stage_name: 'Prepare MaaS',
+    stage: {
+      withCredentials([
+        string(
+          credentialsId: "dev_pubcloud_username",
+          variable: "PUBCLOUD_USERNAME"
+        ),
+        string(
+          credentialsId: "dev_pubcloud_api_key",
+          variable: "PUBCLOUD_API_KEY"
+        ),
+        string(
+          credentialsId: "dev_pubcloud_tenant_id",
+          variable: "PUBCLOUD_TENANT_ID"
+        )
+      ]){
+        dir("rpc-gating/playbooks"){
+          pyrax_cfg = common.writePyraxCfg(
+            username: env.PUBCLOUD_USERNAME,
+            api_key: env.PUBCLOUD_API_KEY
+          )
+          withEnv(["RAX_CREDS_FILE=${pyrax_cfg}"]){
+            common.venvPlaybook(
+              playbooks: ['aio-maas-entities.yml'],
```

Comment:
The aio-maas-entities.yml and aio-maas-cleanup.yml really only need to be run on a multi-node, as an AIO won't need these entities created (and will already have its own entity created at instance creation time).  I don't think you need to fix this now but this is more a note so we remember to add some logic to deal w/ this later.